src/beyond_identity/api/common/token.rs

```diff
@@ -21,7 +22,7 @@ struct StoredToken {
     application_id: String,
 }
 
-pub async fn get_beyond_identity_api_token(
+pub async fn token(
```

Comment:
Can we move AuthMiddleware in this file and make this function module-visible?

```suggestion
async fn token(
```